contracts/MarketMaker.sol

```diff
@@ -54,43 +59,70 @@ contract MarketMaker is Ownable, IERC1155TokenReceiver {
         collateralToken = _collateralToken;
         conditionId = _conditionId;
         fee = _fee;
+
+        require(collateralToken.transferFrom(tx.origin, this, _funding) && collateralToken.approve(pmSystem, _funding));
+        uint[] memory partition = generateBasicPartition();
+        pmSystem.splitPosition(collateralToken, bytes32(0), conditionId, partition, _funding);
+        funding = _funding;
+
         stage = Stages.MarketCreated;
+        emit AutomatedMarketMakerCreated(funding);
     }
 
     function calcNetCost(int[] outcomeTokenAmounts) public view returns (int netCost);
 
     /// @dev Allows to fund the market with collateral tokens converting them into outcome tokens
     /// @param _funding Funding amount
-    function fund(uint _funding)
+    function changeFunding(int _funding)
         public
         onlyOwner
-        atStage(Stages.MarketCreated)
+        atStage(Stages.MarketPaused)
     {
```

Comment:
Right now this is a noop if `fundingChange` is zero. Should the method revert instead?

Reply:
I wasn't really sure about this, should it revert? On a purely technical scale, I don't see much benefit. 

Doesn't cost much to throw a revert in there though, and creates some clarity in the revert message. I don't see much difference. I'll throw the revert in there, for clarity.